evaluate.py

```diff
@@ -77,70 +86,75 @@ def evaluate(test_csvs, create_model, try_loading):
             log_error('Checkpoint directory ({}) does not contain a valid checkpoint state.'.format(FLAGS.checkpoint_dir))
             exit(1)
 
-        logitses = []
-        losses = []
-        seq_lengths = []
-        ground_truths = []
+        def run_test(init_op, dataset):
+            logitses = []
+            losses = []
+            seq_lengths = []
+            ground_truths = []
 
-        print('Computing acoustic model predictions...')
-        bar = progressbar.ProgressBar(widgets=['Steps: ', progressbar.Counter(), ' | ', progressbar.Timer()])
+            bar = progressbar.ProgressBar(prefix='Computing acoustic model predictions | ',
```

Comment:
How about having the same no-progressbar fallback here?
Which raises the question, if the whole progressbar/log thing shouldn't get centralized.

Reply:
Can I do that in a follow-up PR? I'll open an issue to keep track of it.

Reply:
OK

Reply:
https://github.com/mozilla/DeepSpeech/issues/2043

Reply:
Pushed another commit to this PR handling this since the PR was still open.